<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>realTimeProcessClass</class>
 <widget class="QMainWindow" name="realTimeProcessClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>599</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>realTimeProcess</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,1,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>点云配准算法选取</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="selectRegister">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>GICP</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>改进GICP</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ICP</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>NICP</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>non-linearICP</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>FPFH-SAC</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4pcs</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>k4pcs</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>NDT</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="startRegister">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>开始配准</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>点云后处理算法选取</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="selectPostProcess">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>ISS</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Random</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>voexl</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Curvature</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>mesh</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="startPostProcess">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>开始后处理</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>r</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="rSlider">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>g</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="gSlider">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>b</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="bSlider">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="changeBackground">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>背景颜色设定</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QPushButton" name="clearPointCLoud">
          <property name="text">
           <string>清空点云</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveCurrentPointCloud">
          <property name="text">
           <string>保存可视窗口文件</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="enterSystem">
          <property name="text">
           <string>实时三维点云处理系统</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QVTKOpenGLNativeWidget" name="openGLWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>770</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionloadSrcFile"/>
    <addaction name="actionloadTarFile"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>拼接</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>后处理</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>其他操作</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionloadSrcFile">
   <property name="text">
    <string>loadSrcFile</string>
   </property>
  </action>
  <action name="actionloadTarFile">
   <property name="text">
    <string>loadTarFile</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLNativeWidget</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">qvtkopenglnativewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="realTimeProcess.qrc"/>
 </resources>
 <connections/>
</ui>
